docs/src/DeveloperDocumentation/new_developers.md

```diff
@@ -4,67 +4,74 @@ This document is meant to get new developers started. It will not go into depth
 of programming in Julia or working with git, as there are far better resources
 on these things online.
 
-!!! note "Pay attention to your GitHub notifications!"
-    Once you open a pull request on GitHub you will receive feedback, comments,
-    and questions on GitHub. So please pay attention to your GitHub
-    notifications.
+!!! note
+    Please find the technical details of the development process in the Oscar
+    [Way of Working](@ref).
 
 ## Oscar - what is the idea
 
-Oscar is the innovative, next generation Computer Algebra System. The ultimate goal for Oscar is to compete with (and ideally beat) Magma and Sage in our areas of expertise. Oscar should be accessable, even for the youngest student who is familiar with these objects. Oscar should follow general mathematical conventions to support the widest possible range of applications.
+Oscar is the innovative, next generation Computer Algebra System. The ultimate
+goal for Oscar is to compete with (and ideally beat) Magma and Sage in our
+areas of expertise. Oscar should be accessible, even for the youngest student
+who is familiar with these objects. Oscar should follow general mathematical
+conventions to support the widest possible range of applications.
 
 ## Oscar and Julia
 
 Oscar is written in Julia, but is not Julia, nor can it be. 
- Some examples to illustrate what that means:
-  Julias matrices are (C) arrays (of arbitrary dimension), parametrized by
-  the type of the entries (apart from banded, sparse, ... special matrices).
-  In the numerical world, the type mostly defines the representation of an object
-  - double and variations
-  - complex
-  - BigFloat
-  - Int
-  - BigInt
-
-  In algebra, this is either not true or terribly inefficient (or impossible)
-  Take
-    Z/nZ integers modulo n, and matrices over it
+Some examples to illustrate what that means:
+Julia's matrices are (C) arrays (of arbitrary dimension), parameterized by
```

Comment:
I wonder if many people will understand the `(C)` bit, esp. if they are not familiar with the `C` language in the first place. Perhaps the intent was to convey this?
```suggestion
Julia's matrices are flat arrays (of arbitrary dimension), parameterized by
```
Or perhaps it should just be dropped?

Reply:
I dropped it, I didn't understand it.